include/pybind11/attr.h

```diff
@@ -551,6 +576,16 @@ struct process_attribute<base<T>> : process_attribute_default<base<T>> {
     static void init(const base<T> &, type_record *r) { r->add_base(typeid(T), nullptr); }
 };
 
+/// Process a custom base attribute
+template <typename T>
+struct process_attribute<custom_base<T>> : process_attribute_default<custom_base<T>> {
+    static void init(const custom_base<T> &b, type_record *r) {
+        r->add_base(typeid(T), b.fn);
+        // TODO: rename this to 'nonsimple'?
+        r->multiple_inheritance = true;
```

Comment:
You really mean simple_ancestors=False, right?

Reply:
Can't set that here, but that's what setting multiple_inheritance causes. See the TODO comment. :)